pkgs/build-support/ocaml/dune.nix

```diff
@@ -31,12 +31,13 @@ stdenv.mkDerivation ({
     runHook postInstall
   '';
 
-} // (builtins.removeAttrs args [ "minimalOCamlVersion" ]) // {
+} // (builtins.removeAttrs args [ "minimalOCamlVersion" "minimumOCamlVersion" ]) // {
 
   name = "ocaml${ocaml.version}-${pname}-${version}";
 
   nativeBuildInputs = [ ocaml Dune findlib ] ++ nativeBuildInputs;
 
   meta = (args.meta or {}) // { platforms = args.meta.platforms or ocaml.meta.platforms; };
-
-})
+})) // {
+  inherit (args) minimumOCamlVersion minimalOCamlVersion;
```

Comment:
If at all, it should be done via the `passthru` argument to `mkDerivation`. I think, though in practice we can leave it as is (or alternative drop the `removeAttrs`?).

Reply:
Ofborg says it works if passing both arguments directly through mkDerivation, I did tried concatenating the mkDerivation result to both arguments basically to avoid rebuilding the packages but it seems that doesn't matter.